### DISCUSSION TITLE
画饼充饥，一年前画的大饼

### DISCUSSION THREAD

**Discussion by Demonese** [Legacy-LuaSTG-Engine/LuaSTG-Sub#39] [Ideas | 点子王]
人要有梦想，要有目标，万一实现了呢？

* [x] `LuaSTG Sub (-)` Windows 版，正在积极开发
* [ ] `LuaSTG And (&)` Android 版，即 LuaSTG Android
* [ ] `LuaSTG Not (!)` Linux 版
* [ ] `LuaSTG Xor (^)` Apple 版
* [ ] `LuaSTG Mul (*)` 终极目标，跨平台版，即 LuaSTG Multi-Platform

**Comment by KaleiAlma** [comment]
Should I rename Evo to Mul then? Though, it doesn't support android... maybe someday.

DeepL:

那我是不是应该把 Evo 更名为 Mul？虽然它不支持安卓系统......也许有一天会支持的。

**Reply by Demonese** [reply]
Evo is cooler. :)
